### ISSUE TITLE
Missing county data...

### ISSUE THREAD

**Issue by adanecito** [CSSEGISandData/COVID-19#800]
A week ago the report had county level data like Kings County in Washington State. That is one of the USA highest confirmed county infections and deaths yet it no longer shows up. If you have decided to only report state level can you point us to the web service (restful hopefully) that we can use instead of these higher level reports?
Thanks!

**Comment by stevenqzhang** [comment]
+100. Trying to do some visualizations for whether the logy confirmed cases curve is bending (i.e. is exponential growth slowing) at the county level (Bay Area and King County), but unable to do so due to this data issue :(

https://www.dropbox.com/s/3iv1zpl8pi70bny/Screenshot%202020-03-15%2013.19.07.png?dl=0

**Comment by JimBudde** [comment]
Not sure we will be getting County Level data back since amount of effort to track at that level is a bit overwhelming for the folks at JH.  Yes, it would be nice from a situational awareness perspective.  If private labs report it to that level to CDC, that would be great but I am not counting on it.

**Comment by stevenqzhang** [comment]
Ah I see fair enough! I asked around on twitter to see if anyone has this data, will update here if I find any!

https://twitter.com/steven_q_zhang/status/1239291777560895488
https://twitter.com/steven_q_zhang/status/1239289889146863616

**Comment by tfors** [comment]
@stevenqzhang Check the U. Va data.  They're still providing county-level data for US states:

http://nssac.bii.virginia.edu/covid-19/dashboard/

**Comment by stevenqzhang** [comment]
@tfors  thanks! However, it's not time series though? Only the most recent data it seems

**Comment by adanecito** [comment]
There are usually ways each state health department communicates these numbers to the federal government like they do say environmental data that ends up as part of government level Open Data.
That data is usually stored in a database and Restful services are provided using an API to get it. Have you all not wondered about wildfire maps which are quite detailed and used to be provided via GeoMac but that as of this week is no longer available from the government.

**Comment by adanecito** [comment]
If time series has the data for China stopped? I live 33 miles away from Kings County and the virus still grows and the CDC still links to to that web page. Odd thing is I am a John Hopkins student in their Computer Science Master's program 4.0 GPA and they have great classes so technology is not an issue.

**Comment by JimBudde** [comment]
@adanecito I doubt very much this is technology issue.  This occurred so quickly that pulling the data together manually made the most sense at the time.  Once the spread reached a certain point, capturing data at the county level manually became a pipe-dream.  Would it be *nice* if we had it at the county level, sure, but tracking at that level is likely now in the hands of each state's public health dept so they can manage state resources as effectively as possible.  
My after-action report take-away, CDC/WHO should probably develop a reporting mechanism to capture this information automatically for the next outbreak.  Who knows, maybe they already have this, I don't know.

**Comment by adanecito** [comment]
Yep I agree. In regards to collection and reporting there has been a system in place for some time that is free and funded by Bill & Melinda Gates. It is named DHIS and used by WHO and they led me to here when I asked about data. CDC mentions it as software they use. https://www.cdc.gov/csels/dhis/index.html

**Comment by Hikmer** [comment]
The amount of errors in this data is stunning.

**Comment by JimBudde** [comment]
Wasn't aware of DHIS, nor have I looked at it yet, but if it is what we would hope from a data reporting perspective then I will reiterate my lessons-learned comment from this event.  Thanks for making me aware.

**Comment by adanecito** [comment]
Good to hear. I needed the county level for a project. I was going to add it to a new update of this:
https://play.google.com/store/apps/details?id=com.myuniportal.android.apps.wildfireaware&hl=en_US

A potential project for my Thesis later for John Hopkins. Kind of Ironic this happened.

**Comment by saralioness** [comment]
We have a team that has been actively working on this independently - see issue #558 

Hope you can join us!

**Comment by JimBudde** [comment]
Thanks Sara!

**Comment by stevenqzhang** [comment]
@saralioness awesome! Looks like there's a data feed available now: https://github.com/CSSEGISandData/COVID-19/issues/558#issuecomment-599224125

@Hikmer  what errors are you talking about?

**Comment by adanecito** [comment]
Hi Sara,
Thanks for this encouraging news! I will look at the data for Washington State since I work for them. If the county data is in the file for tomorrow I can integrate it with my Big Data database and view it.
I can release my app and then add a way for feedback to jhu somehow for accuracy. Just let me know if that will help.
Best Regards.

**Comment by adanecito** [comment]
Ok Sara the number of states and totals look good for Washington State. It has King County in it also is good. I need to load my database then look at the data on my map application then check individual counts. I used today's  data from #558 
Best Regards,
Tony
